crates/bevy_anti_aliasing/Cargo.toml

```diff
@@ -13,6 +13,8 @@ trace = []
 webgl = []
 webgpu = []
 smaa_luts = ["bevy_render/ktx2", "bevy_image/ktx2", "bevy_image/zstd"]
+dlss = ["dep:dlss_wgpu", "bevy_render/dlss"]
+bevy_ci_testing = ["dlss_wgpu?/mock"]
```

Comment:
This feature isn't made to be used to detect running in CI, it shouldn't be used like that

Reply:
Would a separate disable_dlss feature be fine? I'd have to create it at the top-level bevy crate too, so users would see it as well.

Reply:
yup! for me that would be better than reusing a feature that isn't made for that